src/ceph.in

```diff
@@ -1272,8 +1272,8 @@ if __name__ == '__main__':
     except KeyboardInterrupt:
         print('Interrupted')
         retval = errno.EINTR
-    finally:
-        # shutdown explicitly; Rados() does not
-        if cluster_handle:
-            run_in_thread(cluster_handle.shutdown)
-        sys.exit(retval)
+
+    # shutdown explicitly; Rados() does not
+    if cluster_handle:
+        run_in_thread(cluster_handle.shutdown)
+    sys.exit(retval)
```

Comment:
probably it's just moot, but i think it's a better pattern to move the cleanup into `finally` block to emphasis that "it should always be executed". so maybe we can have something like
```py
    try:
        retval = main()
    except KeyboardInterrupt:
        print('Interrupted')
        retval = errno.EINTR
    finally:
        # shutdown explicitly; Rados() does not
        if cluster_handle:
            run_in_thread(cluster_handle.shutdown)
    sys.exit(retval)
```